ts/src/mexc.ts

```diff
@@ -5245,6 +5245,7 @@ export default class mexc extends Exchange {
         const networks = this.safeValue (this.options, 'networks', {});
         let network = this.safeString2 (params, 'network', 'netWork'); // this line allows the user to specify either ERC20 or ETH
         network = this.safeString (networks, network, network); // handle ETH > ERC-20 alias
+        network = this.networkIdToCode (network);
```

Comment:
@sc0Vu it should be the opposite no (networkCodeToId) ? we want to convert it to networkId from the networkCode